templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>What each variable defines should be documented in markdown text above instead of comments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done</p>